apps/indexer/src/lib/constants.ts

```diff
@@ -457,6 +495,7 @@ export const LendingAddresses: Record<DaoIdEnum, Record<string, Address>> = {
   [DaoIdEnum.SCR]: {
     Aave: "0x25718130C2a8eb94e2e1FAFB5f1cDd4b459aCf64",
   },
+  [DaoIdEnum.OBOL]: {},
```

Comment:
is it supposed to have no lending addresses?

Reply:
There is no LendingAddresses coming from research. So... yes.